### ISSUE TITLE
Provide some way of queuing batches

### ISSUE THREAD

**Issue by Daniel-Mietchen** [arthurpsmith/author-disambiguator#118]
Since only about two batches can run in parallel without causing errors, it would be useful to be able to queue batches, e.g. by sending them to QuickStatements even when in OAuth mode.

**Comment by Daniel-Mietchen** [comment]
Here is an overview of my recent batches to highlight why it would be useful to process batches via an orderly queue.
![Screenshot_2020-03-15 Author Disambiguator copy](https://user-images.githubusercontent.com/465923/76707621-1f6d4000-66c7-11ea-80ea-8599d2c00164.png)

**Comment by arthurpsmith** [comment]
I agree this is needed; I've been thinking about how to do it. I think it might be best to limit to just one active batch at a time (per user).

**Comment by arthurpsmith** [comment]
Batches should now be queued - let me know if you run into trouble with this feature!

**Comment by Daniel-Mietchen** [comment]
Seems to work smoothly for the most part, except that "restart batch" functionality is lost, as per #130 .

**Comment by Daniel-Mietchen** [comment]
Another thing is that it would be nice to have "Remove from queue?" (or "Delete batch") functionality also for inactive but unfinished batches.

**Comment by arthurpsmith** [comment]
Yup, I had a bit of tidying up to do. Remove from queue is there now (as is delete if you use the checkboxes on the batches_oauth page). And restart works (puts back in the queue or starts it if there's no batch running and it would be the oldest queued item.

**Comment by Daniel-Mietchen** [comment]
Seems all sorted out now &mdash; nice!
